src/common/utils/api/api.ts

```diff
@@ -140,4 +144,11 @@ export default {
     apiGet<AuthTestResponse>({
       path: authRequiredTest,
     }),
+
+  testPostPermission: async (resourceId: string): Promise<boolean> => {
```

Comment:
Is this post permission specific to resource? If I understood correctly, it means the user has permission to make a post request to a specific resource? So not general postPermission, but resource-based? Though we currently only have posts to resource path?

Reply:
Good point, I'll change the name